tests/clone_test.go

```diff
@@ -840,3 +852,31 @@ var _ = Describe("[Serial]VirtualMachineClone Tests", Serial, func() {
 		})
 	})
 })
+
+func startCloneVM(virtClient kubecli.KubevirtClient, vm *virtv1.VirtualMachine) (*virtv1.VirtualMachine, error) {
+	patch, err := patch.New(patch.WithReplace("/spec/runStrategy", virtv1.RunStrategyAlways)).GeneratePayload()
```

Comment:
I think it is safer and better to use `WithAdd`.

Reply:
Are you sure? I think only replace would work here as /spec/runStrategy exists at that point